atom/browser/web_contents_preferences.cc

```diff
@@ -124,10 +127,10 @@ WebContentsPreferences::WebContentsPreferences(
   SetDefaultBoolIfUndefined(options::kNodeIntegrationInWorker, false);
   SetDefaultBoolIfUndefined(options::kDisableHtmlFullscreenWindowResize, false);
   SetDefaultBoolIfUndefined(options::kWebviewTag, false);
-  SetDefaultBoolIfUndefined(options::kSandbox, false);
-  SetDefaultBoolIfUndefined(options::kNativeWindowOpen, false);
-  SetDefaultBoolIfUndefined(options::kEnableRemoteModule, true);
-  SetDefaultBoolIfUndefined(options::kContextIsolation, false);
+  SetDefaultBoolIfUndefined(options::kSandbox, secure_mode_enabled);
```

Comment:
I think these defaults should remain the same and we should add a new block on line 152 like below

```cpp
if (secure_mode_enabled) {
  SetBool(options:kSandbox, true);
}
```

To make the current defaults explicit and the behavior of secure mode explicit